src/test/java/com/linecorp/armeria/server/http/tomcat/TomcatServiceTest.java

```diff
@@ -44,11 +45,19 @@
 
     private static final Pattern CR_OR_LF = Pattern.compile("[\\r\\n]");
 
+    private static final String SERVICE_NAME = "TomcatServiceTest";
+
+    private Service[] tomcatServices;
+
     @Override
     protected void configureServer(ServerBuilder sb) {
         sb.serviceUnder(
                 "/tc/",
-                TomcatService.forCurrentClassPath("tomcat_service").decorate(LoggingService::new));
+                TomcatServiceBuilder.forCurrentClassPath("tomcat_service")
+                                    .serviceName(SERVICE_NAME)
+                                    .configurator(s -> tomcatServices = s.findServices())
```

Comment:
Somehow tomcatServices = new ArrayList() and tomcatServices.addAll here makes this more readable to me, but maybe just me.